DNN Platform/Library/Services/Scheduling/DNNScheduler.cs

```diff
@@ -54,19 +54,13 @@ public class DNNScheduler : SchedulingProvider
 
         #region Private Methods
 
-        private bool CanRunOnThisServer(string servers)
+        private static bool CanRunOnThisServer(string servers)
         {
-            string lwrServers = "";
-            if (servers != null)
-            {
-                lwrServers = servers.ToLower();
-            }
-            if (String.IsNullOrEmpty(lwrServers) || lwrServers.Contains(Globals.ServerName.ToLower()))
-            {
-                return true;
-            }
-
-            return false;
+            return
+                // Scheduler should run on every server
+                string.IsNullOrWhiteSpace(servers)
+                // Check if scheduler should run on this server
+                || servers.Split(',').Contains(Globals.ServerName, StringComparer.InvariantCultureIgnoreCase);
```

Comment:
Also, do the servers need to be trimmed before comparing (e.g. would "Server1, Server2" be possible, with a space at the front of the second item)?  Perhaps it would be better to split on comma and space, and exclude empty entries.